### ISSUE TITLE
No rounded corners while the window is dragged

### ISSUE THREAD

**Issue by Petross404** [alex47/KDE-Rounded-Corners#1]
It's not the best [gif](blob:https://imgur.com/a8b3e208-89d8-4711-98c5-20490567c91e) ever, but if you look closely at the window edges, you will notice that the window is becoming sharp again.

**Comment by alex47** [comment]
do you have wobbly windows enabled?

**Comment by Petross404** [comment]
> do you have wobbly windows enabled?

Yes I do. I just discovered that disabling this effect is a workaround for this issue.
